doc/user/content/releases/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Releases"
+description: "How Materialize is released"
+disable_list: true
+menu:
+  main:
+    parent: 'about'
+    weight: 10
+---
+
+We are continually improving Materialize with new features and bug fixes. We
+periodically release these improvements to your Materialize Cloud account. This
+page describes the changes in each release and the process by which they are
+deployed.
+
+## Release notes
+
+{{< warning >}}
+The v0.27.0 release notes describe the coalesced changes between v0.26 LTS
+and the currently released version of Materialize Cloud. We will begin
+publishing weekly release notes after general availability.
+{{< /warning >}}
+
+{{< version-list >}}
```

Comment:
If we're doing weekly releases, isn't it overkill to have a dedicated release page for each? Something similar to what we had for the binary releases seems easier to maintain.

Reply:
I'm ambivalent! After a year of releases the page gets pretty unwieldy. E.g. the [old page](https://materialize.com/docs/release-notes/) has so many headings that you can't actually see them all in the sidebar.

Reply:
Isn't the same bound to happen to this table, though? 🫠 I don't have a better suggestion just now, in any case!

Reply:
This table is prepped to have the same scrollable magic that the old versions table does! https://materialize.com/docs/versions/#stable-releases